docs/server/source/cloud-deployment-templates/node-config-map-and-secrets.rst

```diff
@@ -0,0 +1,72 @@
+Configure the Node
+==================
+
+Use the ConfigMap template in ``configuration/config-map.yaml`` file to
+configure the node. Update all the values for the keys in the
+ConfigMaps ``vars``, ``mdb-fqdn``, ``bdb-public-key``, ``bdb-keyring`` and
```

Comment:
This was renamed to `mdb-instance-name`.
Removed the erroneous reference in the docs.